src/styles/_mediaQueries.scss

```diff
@@ -0,0 +1,58 @@
+@media (max-width: 1000px) {
+    .loginBgContainer {
+
```

Comment:
This project is seriously impressive, Leslie! I'm sorry I can't offer much constructive feedback! I really, honestly only have good things to say - it's responsive, it works well, it's beautiful to look at!

Reply:
Thank you so much for looking through so thoroughly and providing comments throughout!! :) I thought my code was pretty messy (the NotesList containing all of the crammed JS must've hurt to look at, haha :( ), so I really, really appreciate you helping go through and reviewing it!!